src/main/resources/db/changelog/changes/13-create-shopping-carts-table.yaml

```diff
@@ -0,0 +1,27 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-shopping-carts-table
+      author: artem_klishch
+      changes:
+        - createTable:
+            tableName: shopping_carts
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: user_id
```

Comment:
as you use `@MapsId ` don't need to add this column

Reply:
++, thank you so much. updated